test/kommander_test.go

```diff
@@ -164,8 +164,8 @@ kubeaddonsRepository:
 		t.Fatal(err)
 	}
 
-	// there is well known bug of kommander not being able to be uninstalled
-	// https://jira.d2iq.com/browse/D2IQ-63395
+	// there is a bug of kommander not being able to be uninstalled
+	// https://jira.d2iq.com/browse/D2IQ-73310
```

Comment:
Im not sure I fully get the changes to testing code. 

We're still working around the issue, thats what https://jira.d2iq.com/browse/D2IQ-73310 is for to fix. Correct?

lets get https://jira.d2iq.com/browse/D2IQ-73310 fixed then. I'd like to make it a blocker. cc @samba @samvantran

Reply:
Yep, the test changes here are all working around the issue so we can get CI passing and get some of these PRs merged